### PR TITLE
Use SystremClock.elapsedRealtime() to measure durations

### PR DESCRIPTION
**Pull request by vatsalsura**
Issue #2318 Used `SystremClock.elapsedRealtime()` to measure time durations between two events instead of `System.currentTimeMillis`

### CODE REVIEW

File: k9mail/src/main/java/com/fsck/k9/service/MailService.java [hunk 1]
```diff
@@ -88,7 +87,8 @@ public void onCreate() {
 
     @Override
     public int startService(Intent intent, int startId) {
-        long startTime = System.currentTimeMillis();
+        long startTime = SystemClock.elapsedRealtime();
+        long endTime = SystemClock.elapsedRealtime();
```
Comment (Valodim) [inline k9mail/src/main/java/com/fsck/k9/service/MailService.java]: :question:

Comment (vatsalsura) [inline k9mail/src/main/java/com/fsck/k9/service/MailService.java]: Yeah sorry for that. I don't know why I did that. I will revert that change.

**Comment by vatsalsura** [comment]
@Valodim I reverted the changes you told. I am new to Git and Github, so for reverting I pushed another commit. But after that I squashed them together and then forced push it. So did I do the right thing here? And yeah I also found spelling mistake in my commit message so I used `git commit --amend` and then again forced push it. Is this the right way to do it or did I do something wrong here?

**Comment by philipwhiuk** [comment]
Yep that's the expected approach. Looks fine to me.

**Comment by vatsalsura** [comment]
@philipwhiuk What I wanted to ask is if there is any way we can edit the last commit without adding another commit and then squashing them?

File: k9mail-library/src/main/java/com/fsck/k9/mail/power/TracingPowerManager.java [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
-
-import com.fsck.k9.mail.K9MailLib;
-
 import android.content.Context;
 import android.os.PowerManager;
 import android.os.PowerManager.WakeLock;
+import android.os.SystemClock;
 import android.util.Log;
 
+import com.fsck.k9.mail.K9MailLib;
```
Comment (cketti) [inline k9mail-library/src/main/java/com/fsck/k9/mail/power/TracingPowerManager.java]: This change reorders the imports. Please import our [code style settings](https://github.com/k9mail/k-9/wiki/CodeStyle) so Android Studio will use our ordering.

Comment (vatsalsura) [inline k9mail-library/src/main/java/com/fsck/k9/mail/power/TracingPowerManager.java]: @cketti I imported K-9 mail's code style and re-organized imports in all files and also reformatted the code. Please review.

**Comment by cketti** [comment]
Please don't apply the code style to all of the files you touched. This leads to many unrelated changes and makes it hard to find out what you actually changed.

**Comment by vatsalsura** [comment]
@cketti So what do you suggest? Should I undo my last changes of code reformat?

**Comment by cketti** [comment]
Yes. The pull request should only contain your functional changes. Automatically reordered imports because of the added class are fine, too. But no other changes.

**Comment by vatsalsura** [comment]
@cketti @Valodim @philipwhiuk Reverted the re-formatted code changes. Now only imports are re-organized according to K-9 mail's code style. Please review.

**Comment by cketti** [comment]
Thanks.

**Pull request merged**